webapp/portlet/src/main/webapp/WEB-INF/jsp/bannerLogo.jsp

```diff
@@ -73,3 +84,19 @@ document.addEventListener('spaceDetailUpdated', event => {
     </div>
   </div>
 </div>
+<script type="text/javascript">
+  window.require(['SHARED/social-ui-profile'], function(socialProfile) {
+    const labels = {
+      StatusTitle: '<%=res.getString("UserProfilePopup.label.Loading")%>',
+      Connect: '<%=res.getString("UserProfilePopup.label.Connect")%>',
+      Confirm:'<%=res.getString("UserProfilePopup.label.Confirm")%>',
+      CancelRequest: '<%=res.getString("UserProfilePopup.label.CancelRequest")%>',
+      RemoveConnection: '<%=res.getString("UserProfilePopup.label.RemoveConnection")%>',
+      Ignore: '<%=res.getString("UserProfilePopup.label.Ignore")%>',
+      join: '<%=res.getString("UIManageAllSpaces.label.action_join")%>',
+      leave: '<%=res.getString("UIManageAllSpaces.label.action_leave_space")%>',
+      members: '<%=res.getString("UIManageInvitationSpaces.label.Members")%>',
+    };
```

Comment:
Why generating the labels in JSP itself ? it could be handled by Front end UI as well.

Reply:
it is required to be sent while calling the function